ui/app/AppMain.qml

```diff
@@ -195,21 +195,31 @@ RowLayout {
               anchors.topMargin: 0
 
               Rectangle {
+                  id: chatBadge
                   visible: chatsModel.unreadMessagesCount > 0
                   anchors.top: parent.top
                   anchors.left: parent.right
-                  anchors.leftMargin: -10
-                  anchors.topMargin: -5
-                  radius: width / 2
-                  color: Style.current.blue
-                  width: chatsModel.unreadMessagesCount < 10 ? 18 : messageCount.width + 10
-                  height: 18
-                  Text {
-                      id: messageCount
-                      font.pixelSize: chatsModel.unreadMessagesCount > 99 ? 10 : 12
-                      color: Style.current.white
-                      anchors.centerIn: parent
-                      text: chatsModel.unreadMessagesCount > 99 ? "99+" : chatsModel.unreadMessagesCount
+                  anchors.leftMargin: -16
+                  anchors.topMargin: 1
+                  radius: height / 2
+                  color: Style.current.white
+                  width: chatsModel.unreadMessagesCount < 10 ? 22 : messageCount.width + 14
+                  height: 22
+
+                  Rectangle {
```

Comment:
I think that instead of doing a circle (rectangle) inside another one, you just can use the `border` property of the Rectangle